spec/latest/core/index.html

```diff
@@ -1287,6 +1287,51 @@ <h3>Client initiated pagination</h3>
       to provide a custom page identifier (i.e. a GUID or a letter)
       instead of a number.</p>
   </section>
+
+  <section>
+    <h3>Extensions</h3>
+    <p>While Hydra Core Vocabulary allows addressing many usage scenarios,
+      not every aspect of API behavior can be covered. This
+      applies especially to querying, resource projection or data structure
+      description. This is due to fact that Hydra is meant to be as light
+      as possible forcing to drop some of the features out of its scope.</p>
+
+    <p>That is why there is a possibility of hinting a client on what kind
+      of extensions may be found or used in the received payloads.
+      After discovering an <i>extension</i> predicate in the API documentation,
+      client can assume additional details are available described with
+      complementary vocabularies.</p>
+
```

Comment:
Add an example hinting usage of some extension, i.e. vocabulary for data querying.